orga/app/models/organization.js

```diff
@@ -18,19 +18,27 @@ export default class Organization extends Model {
   @hasMany('group', { async: true, inverse: null }) groups;
   @hasMany('division', { async: true, inverse: null }) divisions;
 
-  get isSco1d() {
-    return this.type === 'SCO-1D';
+  get isGarIdentityProvider() {
+    return this.isScoAndManagingStudents && this.identityProviderForCampaigns === 'GAR';
+  }
```

Comment:
En soi l'*Organization* n'est pas un *GarIdentityProvider*, mais peut en avoir un, c'est pourquoi il me semble que l'utilisation de `has` plutôt que `is` serait plus adaptée :

```suggestion
  get hasGarIdentityProvider() {
    return this.isScoAndManagingStudents && this.identityProviderForCampaigns === 'GAR';
  }
```